### PR TITLE
Rubocop should inherit from the .tool-versions

### PR DESCRIPTION
**Pull request by plukevdh**
### What was the problem or feature?

Rubymine was complaining that the 3.X was utilizing the `3.0` ruleset rather than the `3.1` ruleset, thereby flagging some of our new usage of `var = 1 ; func_call(var:)` 

### What was the solution?

Rubocop inherits the version from .tool-versions or a Gemfile/gemspec specification. No need to specify: Drop the specification from the default config.

- [x] Security Impact has been considered (if yes please describe)
- [x] Network Impacts have been considered (if yes please describe)

### Any additional work needed as a result of merging this PR (deploy steps, other PRs, etc.)?

Update Lambda's version

### CODE REVIEW

**Comment by plukevdh** [comment]
Double check me on this one, pls 🙏🏻

File: default.yml [hunk 1]
```diff
@@ -9,8 +9,6 @@ inherit_gem:
   standard: config/base.yml
   rubocop-rspec: config/default.yml
   rubocop-gitlab-security: config/default.yml
-AllCops:
-  TargetRubyVersion: 3.X
```
Comment (jedschneider) [inline default.yml]: sorry, i thought I redacted that before merging. thanks!

**Review by jedschneider — APPROVED** [review]

**Comment by jedschneider** [comment]
there is already a 0.2.0 tag so we may need to bump to .1 https://github.com/highwingio/livery/tree/v0.2.0

I bumped the release but did not update the VERSION constant. what do i know about opensource 🤷

File: lib/livery/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Livery
-  VERSION = "0.1.0"
+  VERSION = "0.2.0"
```
Comment (jedschneider) [inline lib/livery/version.rb]:
```suggestion
  VERSION = "0.2.1"
```
per my top level comment

**Pull request merged**